TestPlaybooks/playbook-PhishlabsIOC_EIR-Test.yml

```diff
@@ -0,0 +1,349 @@
+id: PhishlabsIOC_EIR-Test
```

Comment:
add a task for `FetchFromInstance` in order to test the fetch-incident mechanism